spec/client_spec.rb

```diff
@@ -30,7 +30,7 @@
       }.to raise_error(Acme::Client::Error::BadNonce)
     end
 
-    it 'retry on bad nonce with bad_nonce_retry option', vcr: { cassette_name: 'nonce_retry' } do
+    it 'retry on bad nonce with bad_nonce_retry option', vcr: { cassette_name: 'nonce_retry' }, pending: "Is NONCE required for new_account?" do
```

Comment:
@cpu I can probably figure this out myself by parsing the RFC, but I wonder if you have a quick answer to this question.

As a context, I am trying to determine if this is a good test case for the nonce or if I should pick another endpoint.

Reply:
@weppos `newAccount` does require a nonce. I believe all of the POST requests in RFC 8555 (including the POST-as-GET variant) require a nonce.

Reply:
Thanks!